scenarios/hypertension/hypertension_contactable_overdue_patients.feature

```diff
@@ -0,0 +1,262 @@
+Feature: Contactable Overdue Patients
+  ___
+  As a tester
+  Given I have some "overdue" tracked entity instances with contact information in the system
+  When I export analytics
+  Then I should see the "HTN - Contactable overdue patients" program indicator data as expected
+  ___
+
+  Scenario: All patients must have a patient phone number
+    Given I am signed in as a user with role "Superuser"
+    And I have access to an organisation unit at level 5
+    And I register that organisation unit for program "Hypertension & Diabetes"
+
+    Given I create a new TEI on "7_MonthsAgo" for this Facility with the following attributes
+      | GEN - Given name                      | Callable     |
+      | GEN - Family name                     | Patient      |
+      | GEN - Sex                             | MALE         |
+      | HTN - Does patient have hypertension? | YES          |
+      | HTN - Does patient have diabetes?     | YES          |
+      | GEN - Date of birth                   | 32           |
+      | Address (current)                     | Rose Gardens |
+      | Patient Phone Number                  | 345672781624 |
+      | District                              | KOLARA       |
+      | HTN - Consent to record data          | true         |
+      | HTN - NCD Patient Status              | ACTIVE       |
+    And That TEI has a "Hypertension & Diabetes visit" event on "7_MonthsAgo" with following data
+      | Systole  | 142 |
+      | Diastole | 95  |
+    And That TEI has a "Hypertension & Diabetes visit" event scheduled for "6_MonthsAgo"
+
+    And I create a new TEI on "7_MonthsAgo" for this Facility with the following attributes
+      | GEN - Given name                      | Unreachable  |
+      | GEN - Family name                     | Patient      |
+      | GEN - Sex                             | MALE         |
+      | HTN - Does patient have hypertension? | YES          |
+      | HTN - Does patient have diabetes?     | YES          |
+      | GEN - Date of birth                   | 32           |
+      | Address (current)                     | Rose Gardens |
+      | District                              | KOLARA       |
+      | HTN - Consent to record data          | true         |
+      | HTN - NCD Patient Status              | ACTIVE       |
+    And That TEI has a "Hypertension & Diabetes visit" event on "7_MonthsAgo" with following data
+      | Systole  | 142 |
+      | Diastole | 95  |
+    And That TEI has a "Hypertension & Diabetes visit" event scheduled for "6_MonthsAgo"
+
+    When I wait for 1 second
+    And I export the analytics
+
+    Then The value of "PI":"HTN - Contactable overdue patients" with period type "Months" should be
+      | thisMonth    | 1 |
+      | 1_MonthAgo   | 1 |
+      | 2_MonthsAgo  | 1 |
+      | 3_MonthsAgo  | 1 |
```

Comment:
Should the test be checking for two patients, as there are two overdue patients ?